### DISCUSSION TITLE
Question about bus angle in TDS

### DISCUSSION THREAD

**Discussion by jinningwang** [CURENT/andes#335] [Q&A] [answered]
Hi Hantao,

I have a question about bus angle in the TDS simulation.
I see that the bus angle used now is the relative angle. If I need absolute bus angles, will it be safe to use `a = w * t` as the absolute angle of the slack bus?

Regards,
Jinning

**Comment by cuihantao** [comment, marked as answer]
There isn’t any “absolute bus angle”. All bus angles at the steady state or during TDS are relative to the Slack angle, an implicit reference.

In the equation of a =wt, you confused phasor with waveform. The implicit reference is just a constant value, which is the a0 of Slack being typically zero. After a disturbance, the reference is not equal to the bus angle where the Slack is connected.

Regards,
Hantao Cui

**Reply by jinningwang** [reply]
Many thanks for your reply! I misunderstood the concept of the bus angle.
